content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md

```diff
@@ -38,7 +38,7 @@ kubectl taint nodes node1 key1=value1:NoSchedule
 places a taint on node `node1`. The taint has key `key1`, value `value1`, and taint effect `NoSchedule`.
 This means that no pod will be able to schedule onto `node1` unless it has a matching toleration.
 
-To remove the taint added by the command above, you can run:
+To remove the taint added by the command above, you can run (notice the - at the end of the taint):
```

Comment:
I think this extra line is not required. Current text is self sufficient to describe it.